metadata-load/src/main/java/uk/ac/ebi/ampt2d/metadata/importer/ObjectsImporter.java

```diff
@@ -142,11 +184,88 @@ public Analysis importAnalysis(String accession) {
     protected abstract Analysis extractStudyFromAnalysis(AnalysisType analysisType, Analysis analysis);
 
     public ReferenceSequence importReferenceSequence(String accession) {
+        ReferenceSequence referenceSequence = null;
```

Comment:
As [suggested](https://github.com/EBIvariation/metadata-ws/pull/50#discussion_r291851080) by @iyangar, public/protected/private methods should be grouped separately

Reply:
@iyangar Fixed in https://github.com/EBIvariation/metadata-ws/pull/55/commits/36fe3bd1e265c04d1d918bae849b41d92dcfc9bf

@cyenyxe By the way, do you agree with such an ordering?

Reply:
Per @cyenyxe [comment](https://github.com/EBIvariation/metadata-ws/pull/50#discussion_r292030417) re-ordered methods so that private method follows the public method which first uses it.